src/settings.cpp

```diff
@@ -31,13 +31,15 @@ Settings::Settings(QObject *parent) :
     const QString settings_path =
         QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation)
         + "/" + QCoreApplication::applicationName() + ".conf";
+    QFile settings_file(settings_path);
 
     // Migrate old settings and delete them
     const QString old_settings_path =
             QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) +
             "/harbour-quickddit/Quickddit.conf";
```

Comment:
Fro me the old settings path was $XDG_CONFIG_HOME/harbour-quickddit/harbour-quickddit.conf.

Reply:
That is strange, I made a backup of the settings before which I still have, so I was able to double-check just now and it was Quickddit.conf for me, but it's possible I had the current version in the Jolla store create that by running it outside of Sailjail. `StandardPaths::ConfigLocation` is the same as `$XDG_CONFIG_HOME` though.

Looking at the code, before commit 9da1be80b286fc50384aea972c8700f5c4d6226a, it looks like it should be what you have since it's still `app->setApplicationName("harbour-quickddit");` then.

I guess I can check for both? Or should I just change it to harbour-quickddit.conf?

Reply:
Hm it depends, I think checking for the old name should be sufficient but if you want to be extra cautious check for both.

Reply:
Yeah, looking over this again, I think you are quite right. I changed the old settings location.